src/Microsoft.SqlTools.ServiceLayer/EditData/SmoEditMetadataFactory.cs

```diff
@@ -86,6 +86,10 @@ public EditTableMetadata GetObjectMetadata(DbConnection connection, string[] obj
                     throw new ArgumentOutOfRangeException(nameof(objectType), SR.EditDataUnsupportedObjectType(objectType));
             }
 
+            // Filter out dropped ledger columns from the list of columns to be returned
+            //
+            smoResult.Columns.ClearAndInitialize("[(@IsDroppedLedgerColumn=0)]", new [] {nameof(Column.IsDroppedLedgerColumn), nameof(Column.DataType)});
```

Comment:
>.ClearAndInitialize [](http://example.com/codeflow?start=29&length=19)

Move this after the check for Existing State.
No need to include IsDroppedLedgerColumn in the property list parameter either, just in the filter